mdm/mdm/unmarshal_proto.go

```diff
@@ -163,6 +163,10 @@ func protoToCommand(pb *mdmproto.Command) *Command {
 			ManifestURL:                    nilIfEmptyString(pbc.GetManifestUrl()),
 			ManifestURLPinningCerts:        pbc.GetManifestUrlPinningCerts(),
 			PinningRevocationCheckRequired: nilIfFalse(pbc.GetPinningRevocationCheckRequired()),
+			ManagementFlags:                mgmt,
```

Comment:
The `mgmt` and `configuration` variables are still undefined in this function, you'll have to do the conversion here.

Reply:
ManagementFlags:       int64(zeroIntIfNil(cmd.InstallApplication.ManagementFlags)),